website/docs/docs/cloud/dbt-cloud-ide/git-commit-signing.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Git commit signing"
+description: "Learn how to sign your Git commits when using the IDE for development."
+sidebar_label: Git commit signing
+---
+
+# Git commit signing <Lifecycle status="Enterprise" />
+
+To improve identity protection/impersonation and enhance security, you can sign your Git commits when pushing them to the repository. Using your signature, a Git provider can cryptographically verify it and mark the commit as "verified", giving people increased confidence about the origin of the commit. 
```

Comment:
same comment as below - just "prevent impersonation"

Reply:
Update din this commit: https://github.com/dbt-labs/docs.getdbt.com/pull/6324/commits/73d097b0d7784ef3645b0e715ee364184aaf2471